cdap-docs/_common/common-build.sh

```diff
@@ -127,9 +113,9 @@ function usage() {
   cd $PROJECT_PATH
   PROJECT_PATH=`pwd`
   echo "Build script for '${PROJECT_CAPS}' docs"
-  echo "Usage: ${SCRIPT} < option > [source test_includes]"
+  echo "Usage: ${SCRIPT} <option> [source]"
```

Comment:
How about "action" since it's not really an option (or optional) and it really describes the action you want the script to take.